### PR TITLE
Podman-remote push can support --format 

### PR DESCRIPTION
**Pull request by rhatdan**
Fix man page to document podman push --format fully.

Also found that push was not handling the tlsverify so fixed this.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/source/markdown/podman-push.1.md [hunk 1]
```diff
@@ -90,8 +90,9 @@ solely for scripting compatibility.
 
 #### **--format**, **-f**=*format*
 
-Manifest Type (oci, v2s1, or v2s2) to use when pushing an image to a directory using the 'dir:' transport (default is manifest type of source)
-Note: This flag can only be set when using the **dir** transport. (Not available for remote commands)
+Manifest Type (oci, v2s2, or v2s1) to use when pushing an image.
+Note: The **--format** flag can only be set when using the **dir** and
+**docker** transports. (Not available for remote commands)
```
Comment (mheon) [inline docs/source/markdown/podman-push.1.md:95]: Was this supposed to change the "Not available for remote commands" bit as well?

Comment (rhatdan) [reply docs/source/markdown/podman-push.1.md:95]: Nope. Although It now would work for them.

Comment (vrothberg) [reply docs/source/markdown/podman-push.1.md:95]: If it works for remote, can we drop the comment as well?

Comment (vrothberg) [reply docs/source/markdown/podman-push.1.md:95]: I think we should drop the "Note" entirely.  It also works with the `containers-storage` transport.  Other, such as `docker-archive` or `oci` or `oci-archive` will change the format implicitly.

Comment (rhatdan) [reply docs/source/markdown/podman-push.1.md]: Dropped and added support for --format to podman-remote.

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**